aman/src/pages/Dashboard.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+
+const Dashboard = () => {
+  const navigate = useNavigate();
+
+  const handleClick = () => {
+    localStorage.removeItem("jwtToken");
+    navigate("/");
+  };
+  return (
+    <div className="flex flex-col justify-center items-center">
+      <h1 className="text-red-800 font-extrabold  text-5xl mt-56">
```

Comment:
check here if there's additional space in classname.

Reply:
Done